src/NinjaTools.FluentMockServer.API/Proxy/Evaluation/Evaluators/HttpBodyEvaluator.cs

```diff
@@ -0,0 +1,42 @@
+using System.IO;
+using Microsoft.AspNetCore.Http;
+using NinjaTools.FluentMockServer.API.Extensions;
+
+namespace NinjaTools.FluentMockServer.API.Proxy.Evaluation.Evaluators
+{
+    public class HttpBodyEvaluator : EvaluatorBase
+    {
+        /// <inheritdoc />
+        protected override void EvaluateMember(EvaluationContext context)
+        {
+            if (context.EnsureNotNull(context.Matcher, context.HttpContext.Request.Body) is {} m)
+            {
+
+                var request = context.HttpContext.Request;
+
+                var bodyKind = context.Matcher.BodyKind;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'bodyKind'.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4802644)